packages/variants/src/variants.ts

```diff
@@ -15,10 +16,32 @@ function makeThemeFromPartials(
 }
 
 /**
- * A variant where the background soft shade of the neutral color. Most other colors remain unchanged.
+ * Returns the specified variant theme for the given theme.
+ * IMPORTANT: Never generate a variant from a variant.
  *
  * @export
- * @param {IPartialTheme} theme the theme for which to build a variant for
+ * @param {IPartialTheme} theme the theme to build a variant for
+ * @param {VariantThemeType} variant the variant type designation
+ * @returns {ITheme} the variant theme
+ */
+export function getVariant(theme: ITheme, variant: VariantThemeType): ITheme {
```

Comment:
ooh just noticed, have `theme` here be IPartialTheme
for the `default` case below you'll then need to return createTheme(theme) in order to turn the IPartialTheme into a full ITheme

Reply:
Good catch @phkuo